build.sbt

```diff
@@ -20,6 +20,7 @@ lazy val sslConfigCore = project.in(file("ssl-config-core"))
   .settings(osgiSettings: _*)
   .settings(
     name := "ssl-config-core",
+    mimaPreviousArtifacts := Set("com.typesafe" %% "ssl-config-core" % "0.2.3"), // "sbt mimaReportBinaryIssues"
```

Comment:
Needs to be empty for Scala 2.13:

https://travis-ci.org/lightbend/ssl-config/jobs/417340678#L1126